src/microengine_utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import constants
+from . import datadog
```

Comment:
This may be not needed. `from microengineutils import constants` should work without it.